topics/single-cell/tutorials/scrna-seurat-pbmc3k/tutorial.md

```diff
@@ -0,0 +1,1753 @@
+>            - *"Size of text above colour bar"*: `3.5`
+>            - *"Angle of text above colour bar"*: `60`
+>    - *"Change size of plot"*: `Yes`
+>        - *"Width of plot in pixels"*: `4100`
+>        - *"Height of plot in pixels"*: `4100`
+>
+{: .hands_on}
+
+<div class='Separate-Preprocessing-Steps' markdown='1'>
+
+![Heatmap showing blocks of higher expression for the top 10 markers in the clusters they are markers for. Most cells in each cluster express the markers for that cluster but the patterns are stronger for some genes than others. Some cells outside the clusters also express the markers. For example the CD8T cells show some expression of the top markers for CD4NaiveT and CD4MemoryT cells, although expression of their own top markers is much higher.](../../images/scrna-seurat-pbmc3k/seurat_DoHeatmap_TopPositiveMarkers.png "Heatmap showing expression of the top 10 markers for each cluster")
+
+</div>
+
+<div class='SCTransform' markdown='1'>
+
+![Heatmap showing blocks of higher expression for the top 10 markers in the clusters they are markers for. Most cells in each cluster express the markers for that cluster but the patterns are stronger for some genes than others. Some cells outside the clusters also express the markers. For example, the different T cell subtypes often show high expression of the top markers for other T cell groups.](../../images/scrna-seurat-pbmc3k/seurat_DoHeatmap_TopPositiveMarkers_SCT.png "Heatmap showing expression of the top 10 markers for each cluster")
+
+</div>
+
+Looking at these plots, we can see that there are clear associations between certain genes and clusters. In both cases, we can also see that there is variation in the expression level of these genes within the associated clusters, with some cells expressing little or none of the gene. We can also see cells outside of the associated clusters that are expressing the genes. This is true for both the known cell type markers and the markers we identified through our DE analysis - even when a statistical test has identified a significant association with one cluster, there are still differences between cells.
+
+Comparing the two plots also shows us why the supervised approach can be faster - we had fewer genes to consider and the patterns can also be a bit clearer. If we know enough about the cells we're trying to identify, then we can make our lives easier by taking the supervised approach. However, it is always a good idea to try multiple approaches when we can, as we'll have more confidence in our results if they both end up giving the same answers!
+
+> <question-title></question-title>
+> 1. Which plot type was best for annotating clusters?
+> 2. Are you happy with this clustering - even for cluster <span class='Separate-Preprocessing-Steps'>0</span><span class='SCTransform'>2</span>0, the CD4NaiveT cells, which had a lot of ribosomal genes in its top markers list?
+> > <solution-title></solution-title>
```

Comment:
Just FYI, I added empty lines above all solution boxes that didnt have one, because otherwise it wasn't rendering well. Will work on fixing that bug on the GTN side, but for now quickest to just add the empty lines